src/core/src/op/lrn.cpp

```diff
@@ -2,22 +2,19 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include "ngraph/op/lrn.hpp"
-
-#include <ngraph/validation_util.hpp>
+#include "openvino/op/lrn.hpp"
 
 #include "itt.hpp"
-#include "ngraph/attribute_visitor.hpp"
-#include "ngraph/op/constant.hpp"
-#include "ngraph/op/multiply.hpp"
-
-using namespace std;
-using namespace ngraph;
+#include "openvino/core/attribute_visitor.hpp"
+#include "openvino/core/validation_util.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/multiply.hpp"
 
-op::LRN::LRN(const Output<Node>& arg, double alpha, double beta, double bias, size_t size)
+namespace ov {
+op::v0::LRN::LRN(const Output<Node>& arg, double alpha, double beta, double bias, size_t size)
```

Comment:
What about nesting in such way?
```suggestion
namespace ov {
namespace op {
namespace v0 {
LRN::LRN(const Output<Node>& arg, double alpha, double beta, double bias, size_t size)
```

Reply:
However I think it'd be better to not expand this PR ;)